index.js

```diff
@@ -0,0 +1,80 @@
+const stream = require('stream');
+const streamMap = require('through2-map');
+
+module.exports = (knex, config) => {
+  const columnName = config.columnName || 'value';
+
+  async function open () {
+    return knex.schema.createTableIfNotExists(config.tableName, function (table) {
+      table.increments();
+      table.jsonb(columnName);
+      table.timestamps(true, true);
+    }).then(() => {
+      if (config._purgeLog) return knex(config.tableName).delete();
+    });
+  }
+
+  async function close () {
+    return Promise.resolve();
```

Comment:
you can just return `true` or anything really, as it's a value, and an `async` function will just resolve to that vaule.